src/main/scala/shine/C/AST/Printer.scala

```diff
@@ -88,7 +88,7 @@ class CPrinter extends Printer {
     case b: Block => printBlock(b)
     case f: ForLoop => printForLoop(f)
     case w: WhileLoop => printWhileLoop(w)
-    case i: IfThenElse => printIfThenElse(i)
+    case i: shine.C.AST.IfThenElse => printIfThenElse(i)
```

Comment:
why is this case different now?

Reply:
I was surprised by this as well. It seems as if the Scala compiler has changed some of its resolution mechanism and was now selecting the wrong `IfThenElse` class.